store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -252,6 +252,12 @@ public boolean doLoad(List<File> files) {
                 continue;
             }
 
+            if (file.length() == 0) {
+                boolean ok = file.delete();
```

Comment:
IMO, It is safer to add a judgment which ensure that it is the last file before deleting it